utils/chain.ts

```diff
@@ -15,6 +15,10 @@ export const getChainNameByPrefix = (prefix: string) => {
   return prefix
 }
 
+export const getChainName = (prefix: Prefix) => {
+  return NAMES[prefix].split(' [Beta]').join('')
```

Comment:
It would be more simple
```suggestion
  return NAMES[prefix].replace(' [Beta]', '')
```

Reply:
yep